build-package.sh

```diff
@@ -411,6 +411,7 @@ _show_usage() {
 	echo
 	echo "Available options:"
 	[ "$TERMUX_ON_DEVICE_BUILD" = "false" ] && echo "  -a The architecture to build for: aarch64(default), arm, i686, x86_64 or all."
+	echo "  -c Continue previous build, skip source extraction and configure step and go straight to termux_step_make."
```

Comment:
https://github.com/termux/termux-packages/blob/946cfd777598c5d319d3c08cda19856859d28d56/build-package.sh#L645-L658

In continue build, `termux_step_configure` will be executed, so if `termux_step_configure` is overridden by `build.sh`, configure step is not skipped.

Reply:
I have no idea what it does. I just add the option in help output which was brought up in a GitHub discussion.